ote/src/cljs/ote/views/register.cljs

```diff
@@ -1,15 +1,25 @@
 (ns ote.views.register
   "OTE registration form page."
   (:require [reagent.core :as r]
-            [cljs-react-material-ui.reagent :as ui]
             [ote.ui.form :as form]
             [ote.app.controller.login :as lc]
             [ote.localization :refer [tr tr-key]]
             [ote.ui.buttons :as buttons]
             [ote.ui.common :as common]
+            [ote.db.user :as user]
+            [ote.ui.notification :as notification]))
 
-            [ote.db.user :as user]))
 
+(defn token-check
+  [token-info]
+  (println token-info)
+  (if (not (= token-info :token-invalid))
+    (notification/notification
+      {:text (str "Olet liittymässä osaksi palvelua " (:name token-info))
```

Comment:
Would not hurt if you added this to translation files

Reply:
Jussi is going to figure out proper texts for these notifications first.